Marlin/Configuration.h

```diff
@@ -1556,6 +1556,9 @@
 #define PREHEAT_2_TEMP_BED    110
 #define PREHEAT_2_FAN_SPEED     0 // Value from 0 to 255
 
+// Insert a menu for preheating at the top level to allow for quick access
+#define PREHEAT_MENU_ITEM_SHORTCUT
```

Comment:
Why not just use `CUSTOM_USER_MENUS`?

Reply:
Last i checked CUSTOM_USER_MENUS won't pick up the preheat constants saved on eeprom, only from firmware. You would have to upload the firmware again every time you want to change those shortcuts

Reply:
This PR #20327 added a %t to get the ative extruder. I don't think it should be done here too, but that type of dynamic variable substitution would be very useful for `CUSTOM_USER_MENUS`.

Reply:
I'd be very happy with dynamic variables on CUSTOM_USER_MENUS.  Even better if i could see the dynamic value on the command label. e.g. "PLA **_210_** / **_45_**"

Reply:
should a FR be created for this?